src/pcapi/workers/apps_flyer_job.py

```diff
@@ -0,0 +1,69 @@
+import logging
+
+import requests
```

Comment:
```suggestion
from pcapi.utils import requests
```

Cela logge les appels, les temps et statuts de réponse, force à indiquer un timeout, etc.